kratos/solving_strategies/schemes/scheme.h

```diff
@@ -225,192 +191,185 @@ class Scheme
     {
     }
 
-
     /** Destructor.
      */
     virtual ~Scheme()
     {
     }
 
-
-    /*@} */
-    /**@name Operators
-     */
-    /*@{ */
+    ///@}
+    ///@name Operators
+    ///@{
 
     /**
-     * Clone 
+     * @brief Clone method
+     * @return The pointer of the cloned scheme 
      */
     virtual Pointer Clone()
     {
-      return Pointer( new Scheme(*this) );
+        return Kratos::make_shared<Scheme>(*this) ;
     }
 
     /**
-     * Component wise components Get method
+     * @brief Component wise components Get method
+     * @warning Must be defined on the derived classes
+     * @return The local system of components
      */
-
     virtual LocalSystemComponents& GetLocalSystemComponents()
```

Comment:
i think that this function is not used anywhere (I grepped it) and i would remove it. As a different PR though...